content/introduction/getting-started.md

```diff
@@ -483,7 +483,7 @@ class Book < Hanami::Entity
 end
 ```
 
-This class will generate getters and setters for each attribute which we pass to initialize params.
+This class will generate getters and setters for each attribute which we pass to initialize `params`.
```

Comment:
```suggestion
This class will generate getters and setters for each attribute we pass to initialize `params`.
```

Reply:
I don't understand this one. What do you mean? "which" has not been removed in this PR.

Reply:
Yeah, it's the change I suggest. I think it should be removed.